sklearn/sklearn-template/template/trainer/model.py

```diff
@@ -20,34 +20,48 @@
 from sklearn import pipeline
 from sklearn import preprocessing
 
+import numpy as np
+
 from trainer import metadata
 
 
 def get_estimator(flags):
-  classifier = ensemble.RandomForestClassifier()
-
-  # TODO(cezequiel): Make use of flags for hparams
-  _ = flags
+  # TODO: Allow pre-processing to be configurable through flags
+  classifier = ensemble.RandomForestClassifier(**flags)
 
   numeric_transformer = pipeline.Pipeline([
-      ('imputer', impute.SimpleImputer(strategy='median')),
-      ('scaler', preprocessing.StandardScaler()),
+    ('imputer', impute.SimpleImputer(strategy='median')),
+    ('scaler', preprocessing.StandardScaler()),
+  ])
+
+  numeric_log_transformer = pipeline.Pipeline([
```

Comment:
I'm guessing these new numeric transformers improve model performance?
It seems that they are adding new features as well.
Would suggest adding comments to explain their desired effects.

Reply:
Scale (log, square root) transformation and bucketing the numerical attributes are almost standard transformation, so I think it would helpful to include them as a template. I will add some comments here.